py_gtfs_rt_ingestion/lib/batcher.py

```diff
@@ -0,0 +1,97 @@
+import logging
```

Comment:
not sure why github didn't pickup the filepath rename on this, its largely the same, but a touch smaller as we're no longer compressing / uncompressing filepaths, creating event json strings, nor triggering ingestion lambdas as everything is being handled inside of one instance now.